hedgehog/src/Hedgehog/Internal/Gen.hs

```diff
@@ -486,8 +487,10 @@ instance Monad m => Monad (GenT m) where
           runGenT size sk . k =<<
           runGenT size sm m
 
+#if MIN_VERSION_base(4,14,0)
```

Comment:
I thought it should be `4,13,0` https://gitlab.haskell.org/ghc/ghc/wikis/commentary/libraries/version-history

Reply:
That compiled with both ghc 8.6 and 8.8 but may not actually have been correct.

Think I have fixed it now.